osf/models/registrations.py

```diff
@@ -504,6 +504,9 @@ def terminate_embargo(self, forced=False):
             save=True
         )
         self.embargo.mark_as_completed()
+        if self.embargo_termination_approval:
```

Comment:
nit: would prefer to check `self.is_pending_embargo_terminaton`. This version will trigger some unnecessary state machine code in the case of an actual EmbargoTerminationApproval.